sshd-common/src/main/java/org/apache/sshd/common/future/VerifiableFuture.java

```diff
@@ -57,6 +58,18 @@ default T verify(long timeout, TimeUnit unit) throws IOException {
         return verify(unit.toMillis(timeout));
     }
 
+    /**
+     * Wait and verify that the operation was successful
+     *
+     * @param timeout The maximum duration to wait
+     * @return The (same) future instance
+     * @throws IOException If failed to verify successfully on time
+     * @see #verify(long)
+     */
+    default T verify(Duration timeout) throws IOException {
+        return verify(timeout.toMillis());
```

Comment:
In other locations the  code interprets `null` as *forever* - let's maintain this convention (or fix the other code - `ClientChannel#waitFor`)
```java
default T verify(Duration timeout) throws IOException {
    return (duration == null) ? verify() : verify(timeout.toMillis());
}
```